nuxt/pages/PrintLyricsPage.vue

```diff
@@ -0,0 +1,130 @@
+<router>
+  path: /print/:reciterId/:albumId/:trackId
+  name: "print.lyrics"
+</router>
+
+<template>
+  <div>
+    <div class="print__header">
+      <div class="print__header__title">
+        <div class="print__header__title--track--name">
+          {{ track.title }}
+        </div>
+        <div class="print__header__title--track--meta">
+          {{ track.reciter.name }} - {{ track.year }}
+        </div>
+      </div>
+      <v-spacer />
+      <logo-icon class="print__header__logo" />
+    </div>
+    <lyrics-renderer v-if="track.lyrics" class="print__content" :track="track" />
+    <div
+      v-else
+      class="print__content print__content--empty"
+    >
+      We don't have a write-up of this nawha yet.
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+/* eslint-disable dot-notation */
+import Vue from 'vue';
+import LyricsRenderer from '@/components/lyrics/LyricsRenderer.vue';
+import { TrackIncludes } from '@/api/tracks';
+import { Track } from '@/entities/track';
+const LogoIcon = require('@/assets/svg/icon.svg?inline');
+
+interface Data {
+  track: Track | null;
+}
+
+export default Vue.extend({
+  name: 'LyricsPrintLayout',
+
+  layout: 'print',
+
+  components: {
+    LyricsRenderer,
+    LogoIcon,
+  },
+
+  async fetch() {
+    const { reciterId, albumId, trackId } = await this.$route.params;
```

Comment:
No need for `await` on this line. 

```suggestion
    const { reciterId, albumId, trackId } = this.$route.params;
```